src/tests/ftest/util/test_utils_pool.py

```diff
@@ -960,6 +960,26 @@ def get_pool_free_space(self, device="scm"):
             free_space = daos_space["s_free"][1]
         return free_space
 
+    def get_pool_used_space(self, device="scm"):
```

Comment:
Question - why add this if it isn't being used yet?

Reply:
As mentioned in the commit message, it is for future use. I thought it is something that can come in handy. Don't see any harm in adding something like this.

Reply:
It's just extra code to maintain that isn't being used. If something changes to break the function, we won't know until someone tries to use the function

Reply:
It's removed now. IMO having something like get_pool_used_space() where we already have  get_pool_free_space() could have been useful at some point.

Reply:
I agree it could be useful, but I think it's better to add when needed. Thanks!